composer.json

```diff
@@ -3,13 +3,13 @@
   "description": "Postgis extensions for laravel. Aims to make it easy to work with geometries from laravel models",
   "require": {
     "php": ">=7.1",
-    "illuminate/database": "^6.0|^7.0",
+    "illuminate/database": "^6.0|^7.0|^8.0",
     "geo-io/wkb-parser": "^1.0",
     "jmikola/geojson": "^1.0",
     "bosnadev/database": "0.20.*"
```

Comment:
bosnadev/database does not allow illuminate/database 8.x to be installed, so before bosnadev/database this code-change won't work?
Tested it locally now and get the error: `- bosnadev/database 0.20 requires illuminate/database ^5.0|^6.0|^7.0`

Reply:
a PR for that appears to be pending at https://github.com/bosnadev/database/pull/42
That and this here combined seems to work fine for me, but I make very limited use of it (just a few geography fields)

Reply:
Yep, that's my PR to `bosnadev/database`.

I've got two projects in development and am running my fork of `bosnadev/database` with the Laravel 8 upgrade and a fork of `mstaack/laravel-postgis` just with the updated composer requirements for `bosnadev/database` pointing at my fork.

I've had no issues so far (running with Laravel 8, PHP 7.4, PostgreSQL 12).